src/components/Tool/FixSendNewsletter/FixSendNewsletter.tsx

```diff
@@ -156,16 +147,16 @@ const FixSendNewsletter: React.FC<Props> = ({ accountListId }: Props) => {
             </Typography>
           </Box>
         </Grid>
-        {!loading && data && !!numberOfContacts ? (
+        {!loading && data && !!numberOfContactsShowing ? (
           <>
             <StickyButtonHeaderBox mb={0}>
               <Box>
                 <Typography>
                   <Trans
-                    defaults="<i>Showing <bold>{{numberOfContacts}}</bold> of <bold>{{totalCount}}</bold></i>"
+                    defaults="<i>Showing <bold>{{numberOfContactsShowing}}</bold> of <bold>{{totalCount}}</bold></i>"
                     shouldUnescape
                     values={{
-                      numberOfContacts,
+                      numberOfContactsShowing,
```

Comment:
FYI, changing a label requires that it be retranslated in every language. I think it might be better to do this and keep the label the same:

```suggestion
                      numberOfContacts: numberOfContactsShowing,
```

Reply:
ah, good idea